### ISSUE TITLE
Typo in Zipalign section?

### ISSUE THREAD

**Issue by iamktm** [MicrosoftDocs/xamarin-docs#2910]
The section states 

> The follow command will use the **signed** APK and produce a **signed**, zipaligned APK called helloworld.apk that is ready for distribution.

It then goes on to provide the following command: 

> $ zipalign -f -v 4 mono.samples.helloworld-**unsigned**.apk helloworld.apk

The text stating that the file is initially _signed_ but the command having _unsigned_ in the name is confusing. Also, from reading the top of the page I was initially under the impression that zipalign must be performed before signing. However, from reading this section I wasn't so sure anymore. 

Turns out, the correct way to do it depends on [whether you are signing with apksigner or jarsigner](https://developer.android.com/studio/command-line/zipalign). If you're signing with apksigner, zipalign must only be performed _before_ signing. If you're signing with jarsigner, zipalign must only be performed _after_ signing. 

Since the page is recommending that users sign with apksigner, it would be nice to see the zipalign section follow the apksigner method of performing zipalign before signing. In other words, just change the text before the sample command by replacing both of the _signed_ values to _unsigned_. It would also be nice to have a note in the jarsigner section that if a user chooses to sign with jarsigner, signing must happen before performing zipalign (and, perhaps, link to the [zipalign documentation](https://developer.android.com/studio/command-line/zipalign)) .

---
#### Document Details

⚠ *Do not edit this section. It is required for docs.microsoft.com ➟ GitHub issue linking.*

* ID: d6c53ae1-252f-3869-b8c4-4ffb2e927444
* Version Independent ID: d2343aa0-f82b-d2fe-c6a2-b9b47a93df7b
* Content: [Manually Signing the APK - Xamarin](https://docs.microsoft.com/en-us/xamarin/android/deploy-test/signing/manually-signing-the-apk#Zipalign_the_APK)
* Content Source: [docs/android/deploy-test/signing/manually-signing-the-apk.md](https://github.com/MicrosoftDocs/xamarin-docs/blob/live/docs/android/deploy-test/signing/manually-signing-the-apk.md)
* Product: **xamarin**
* Technology: **xamarin-android**
* GitHub Login: @davidortinau
* Microsoft Alias: **daortin**

**Comment by davidbritch** [comment]
Thanks for submitting an issue and we apologize for the delayed response. The requested updates haven't been made since the creation of the issue, and the timeline for resolution may vary based on resourcing, so the issue is being moved to an internal repo to better track it.
